exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -107,6 +107,42 @@
   <p>FOREMAN: ${model.getProfile().getForeman().getAddress()}</p>
   <p>TOTAL FRAGMENTS: ${model.getProfile().getTotalFragments()}</p>
 
+  <#if (model.getOptionList()?size > 0)>
```

Comment:
For older profiles, I assume `model.getOptionList()` returns `null`. How is `null` handled?